features/dependency_injection/autowiring_contexts.feature

```diff
@@ -131,3 +131,48 @@ Feature: Autowiring contexts
             """
         When I run Behat
         Then it should pass
+
+    Scenario: Autowiring a context with test client
+        Given a feature file containing:
```

Comment:
```
Given a working Symfony application with SymfonyExtension configured
And a feature file containing
```

Because you must use CompilerPass from `src/Bundle/DependencyInjection/FriendsOfBehatSymfonyExtensionExtension:process`.